### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by armandocomellas1**
### This project is created to add a mobile menu with javascript to the Portfolio project

Things we have worked on:

- Mobile Menu for Portfolio
- Javascript as a programming language
- DOM Manipulation.

### CODE REVIEW

File: javascript.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ function mobileMenu() {
   document.getElementsByClassName('ham_menu')[2].style.display = 'flex';
 }
 mobileMenu();
+
 function desktopMob() {
   document.getElementsByClassName('name_head')[0].style.display = 'flex';
   document.getElementsByClassName('material-symbols-outlined')[0].style.display = 'flex';
```
Comment (HAMMAS-SALEEM) [inline javascript.js:14]: ### Missing Requirements :m:
missing-req (copy)

- [ ] It's great that you have javascript and you know how you can manipulate your dom. :clap: but you have not implemented all the requirements. Those requirements have been shown in the image above. :arrow_up_small:

File: index.html [hunk 2]
```diff
@@ -174,15 +174,15 @@ <h3 class="styleh3">CONTACT US</h3>
...
+        <img src=".img/Vector.png" alt="GitHub">
+        <img src=".img/dos.png" alt="LinkedIn">
+        <img src=".img/instagram.png" alt="Instagram">
+        <img src=".img/cuatro.png" alt="Medium">
       </div>
       <p class="emails">businesswitharmando@gmail.com</p>
     </footer>
     <p class="division_footer"></p>
   </div>
-  <script src="/javascript.js"></script>
```
Comment (HAMMAS-SALEEM) [inline index.html:186]: ### Your Design
navbar

### Figma Template
### 1. Fix
figma-nav (copy)

### 2. Fix
figma-nav (another copy)

- [ ] It's great that you have implemented a navbar in your markup file. :blush: But the design is not following the Figma template :cry: Please make your design follow the Figma template :+1:

Comment (HAMMAS-SALEEM) [inline index.html:186]: ### Your Design
navbar (copy)

- [ ] You can see in the image that there are some green boxes and red boxes. Green boxes show the correct position for the contents which are in the red boxes. Please make them on the correct position to follow the Figma template :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @armandocomellas1 :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Following HTML, CSS best practices
:heavy_check_mark: Following Project Requirements
:heavy_check_mark: Descriptive PR title
:heavy_check_mark: Descriptive branch name

Good job on implementing the project requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: javascript.js (hunk 1 above)
Comment (armandocomellas1) [reply javascript.js]: Hello! no the menu dissapears

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @armandocomellas1 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -11,29 +11,29 @@
...
   <div class="container">
-      <header class="menucontainer">
-        <h4 class="name_head">John Doe</h4>
-        <nav id="hambur_menu" class="navmenu">
-          <div class="menu_ham_mob" onClick="desktopMob()">
-            <span class="">
+    <header class="menucontainer">
+      <h4 class="name_head">John Doe</h4>
+      <nav id="hambur_menu" class="navmenu">
+        <div class="menu_ham_mob" onClick="desktopMob()">
```
[the remark this answers is not in the document]
Comment (armandocomellas1) [reply index.html:17]: Hello Add correct link menu to the correct section of the portfolio, please don't make me ask for a special review :( I really do work hard but the instructions sometimes are not 100% clear.

**Review by danmainah — APPROVED** [review]
Hi @armandocomellas1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**